livekit_recording.proto

```diff
@@ -34,24 +34,15 @@ message RecordingInput {
 
 message RecordingTemplate {
   string layout = 1;
-  string ws_url = 2;
   // either token or room name required
   string token = 3;
   string room_name = 4;
 }
 
 message RecordingOutput {
-  string file = 1;
-  RecordingS3Output s3 = 2;
```

Comment:
we should leave s3 in there?

Reply:
it's not a RecordingS3Output anymore, just a string below

Reply:
ohhh I see. my bad reading the diff